workloads/statefulset/sampeapp.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: ApplicationConfiguration
+metadata:
+  name: example-appconfig
+spec:
+  components:
+    - componentName: example-sts
```

Comment:
I saw the warning `No newline at end of file`, so how about adding new a new line in this file and those blow :)

Reply:
that's not important, but I'm fine